pydcs_extensions/llhmodpack/llhmodpack.py

```diff
@@ -0,0 +1,139 @@
+# Requires French Pack mod :
+# https://www.reddit.com/r/DCSExposed/comments/m5uje8/insurgent_pack_work_in_progress_impressions/
+#
+from dcs import unittype
+
+from game.modsupport import vehiclemod
+
+
+@vehiclemod
+class HPG_Ural_Isis_Troops(unittype.VehicleType):
+    id = "HPG_Ural_Isis_Troops"
+    name = "*LLH Ural ISIS Troops"
+    detection_range = 0
+    threat_range = 0
+    air_weapon_dist = 0
+
+
+@vehiclemod
+class HPG_Ural_Isis_Covered(unittype.VehicleType):
+    id = "HPG_Ural_Isis_Covered"
+    name = "*LLH Ural ISIS Covered"
+    detection_range = 0
+    threat_range = 0
+    air_weapon_dist = 0
+
+
+@vehiclemod
+class HPGUralIsisFlak(unittype.VehicleType):
+    id = "HPGUralIsisFlak"
+    name = "*LLH Ural ISIS Flak 88"
+    detection_range = 0
+    threat_range = 0
+    air_weapon_dist = 0
+
+
+@vehiclemod
+class HPGUralIsisZU23(unittype.VehicleType):
+    id = "HPGUralIsisZU23"
+    name = "*LLH Ural ISIS ZU23"
+    detection_range = 7500
+    threat_range = 2500
+    air_weapon_dist = 2500
+
+
+@vehiclemod
+class HPG_Toyota_ISIS(unittype.VehicleType):
+    id = "HPG_Toyota_ISIS"
+    name = "*LLH Toyota ISIS"
+    detection_range = 0
+    threat_range = 0
+    air_weapon_dist = 0
+    eplrs = True
+
+
+@vehiclemod
+class HPGToyotaISISDShK(unittype.VehicleType):
+    id = "HPGToyotaISISDShK"
+    name = "*LLH Toyota ISIS DShk"
+    detection_range = 0
+    threat_range = 0
+    air_weapon_dist = 0
+
+
+## FORTIFICATIONS
```

Comment:
Are these vehicles? Beside that they are added to the list of vehicles they dont have a corresponding unit.yaml for them

Reply:
Yes they are vehicles, and they do have corresponding unit.yamls for them